.editorconfig

```diff
@@ -1,6 +1,72 @@
-[*.fs]
-indent_size=4
+
+
+# Default settings:
+# A newline ending every file
+# Use 4 spaces as indentation
+[*]
+insert_final_newline = true
+indent_style = space
+indent_size = 4
+trim_trailing_whitespace = true
+end_of_line = lf
+
+
+[*.{fs,fsi,fsx,config}]
+# https://fsprojects.github.io/fantomas/docs/end-users/Configuration.html
+charset = utf-8
+trim_trailing_whitespace = true
 max_line_length=240
 fsharp_max_infix_operator_expression = 120
 fsharp_max_value_binding_width = 120
-fsharp_max_if_then_else_short_width = 20
\ No newline at end of file
+fsharp_max_if_then_else_short_width = 20
+fsharp_multiline_block_brackets_on_same_column=true
+fsharp_experimental_stroustrup_style=true
```

Comment:
Sorry, `fsharp_multiline_bracket_style = experimental_stroustrup`.
But if you are using this setting, I'd really go with `v6` as there are a ton of improvements made there.
Note that in v6, it is `fsharp_multiline_bracket_style = stroustrup`.